note.py

```diff
@@ -50,23 +60,33 @@ def get_note(note_name):
     if not note_dict:
         return None
 
-    list_checks = ["links", "backlinks"] # db can't store empty lists, fix db
+    list_checks = ["links", "backlinks"]  # db can't store empty lists, fix db
 
     for each in list_checks:
-      if not note_dict[each]:
-        note_dict[each] = []
-    
-    if not note_dict["content"]: # db can't store empty strings, fix db
+        if not note_dict[each]:
+            note_dict[each] = []
+
+    note_content = drive_notes.get(note_name)
+    if note_content:
+        note_content = str(note_content.read(), 'ascii')
+        note_dict["content"] = note_content
+
+    if not note_dict["content"]:  # db can't store empty strings, fix db
         note_dict["content"] = ""
 
     return Note(**note_dict)
 
-
-# update note with new info  
+# update note with new info
 def db_update_note(note: Note):
     note_dict = note.dict()
     note_dict["content"] = bleach.clean(note_dict["content"])
-    notes.put(note_dict, urlsafe_key(note.name))
+
+    drive_notes.put(note_dict["name"], str(note_dict["content"]))
```

Comment:
I think you'll have to put it under the url safe key e.g. `urlsafe_key(note.name)`?